rosetta/invoker/invoker.go

```diff
@@ -94,28 +86,28 @@ func (i *Invoker) Script(height uint64, script []byte, arguments []cadence.Value
 		args = append(args, arg)
 	}
 
-	// look up the current block and commit for the block
+	// Look up the current block and commit for the block.
 	header, err := i.index.Header(height)
 	if err != nil {
 		return nil, fmt.Errorf("could not get header: %w", err)
 	}
 
-	// we initialize the virtual machine context with the given block header so
-	// that parameters related to the block are available from within the script
+	// Initialize the virtual machine context with the given block header so
+	// that parameters related to the block are available from within the script.
 	ctx := fvm.NewContext(zerolog.Nop(), fvm.WithBlockHeader(header))
 
-	// Initialize the read function. We use a shared cache between all heights
+	// Initialize the read function. A shared cache is used between all heights
 	// here. It's a smart cache, which means that items that are accessed often
-	// are more likely to be kept, regardless of height. This allows us to put
+	// are more likely to be kept, regardless of height. This allows putting
```

Comment:
I find these changes really irrelevant. In this case, it doesn't read better or worse, but it just seems like a waste of time to change it.